obras/service/main.py

```diff
@@ -1,12 +1,104 @@
+import os
 import logging
-from flask import Flask
+from flask import Flask,  request, jsonify, json
```

Comment:
El archivo main.py... es unicamente para configurar flask con namespaces blue prints y que classes formaran parte de los endpoints a publicar.

Y
Las clases que se encargan de los endpoints deberan estar contenidas en otro paquete

namespaces --> https://medium.com/ki-labs-engineering/designing-well-structured-rest-apis-with-flask-restplus-part-1-7e96f2da8850

Reply:
Resources <--- las clases encargadas de los endpoints

A resource is a class whose methods are mapped to an API/URL endpoint. We use flask-restplus annotations to define the URL pattern for every such class.